### PR TITLE
Remove security warning (SECURITY-2875) from Multibranch scan webhook plugin

### PR DESCRIPTION
**Pull request by gdisanto**
A new version of the plugin 1.0.11 has been released with a change addressing this security issue.
Please review fix and remove the security warning if the change is good. Related PR: https://github.com/jenkinsci/multibranch-scan-webhook-trigger-plugin/pull/15

Please check as well the pattern, it should enable the warning up to version 1.0.10.

### CODE REVIEW

File: resources/warnings.json [hunk 1]
```diff
@@ -17207,8 +17207,8 @@
     "url": "https://www.jenkins.io/security/advisory/2023-10-25/#SECURITY-2875",
     "versions": [
       {
-        "lastVersion": "1.0.9",
-        "pattern": ".*"
+        "lastVersion": "1.0.11",
+        "pattern": "(1[.]0[.]10|1[.][0].[0123456789])(|[.-].+)"
```
Comment (daniel-beck) [inline resources/warnings.json:17211]:
```suggestion
        "pattern": "(1[.]0[.]10|1[.][0].[0-9])(|[.-].+)"
```

Both are correct, this is just more efficient (and IMO easier to read).

**Review by daniel-beck — APPROVED** [review]
Fix in the plugin looks fine.

File: resources/warnings.json (hunk 1 above)
Comment (gdisanto) [reply resources/warnings.json:17211]: Hi @daniel-beck,
Yes I had made this change locally indeed, I think I have reverted it by mistake. I will commit it.

**Comment by gdisanto** [comment]
Hi @daniel-beck,
Any updates on when this PR will get merged? It has been open for 2 weeks already.

Thanks.

**Comment by daniel-beck** [comment]
@gdisanto Thanks for the reminder!

**Comment by gdisanto** [comment]
> @gdisanto Thanks for the reminder!

Thanks for merging.

**Pull request merged**